Models/TaskModel.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Models;
+
+class TaskModel
+{
+    public function __construct()
+    {
+        $this->pdo = new \PDO('mysql:host=test.local;dbname=Family', 'snuff', 'kyzmi4');
```

Comment:
Please take a look at my project i sent you today. You can read, understand and implement database model from there and make it even better